python/tank/util/shotgun.py

```diff
@@ -398,7 +398,7 @@ def create_sg_app_store_connection():
     return g_app_store_connection
 
 
-def __get_app_store_connection_information():
+def _get_app_store_connection_information():
```

Comment:
yes, please leave it. Can we also do `__get_app_store_proxy_setting` --> `_get_app_store_proxy_setting` ?